base_tier_validation_formula/tests/test_tier_validation.py

```diff
@@ -0,0 +1,169 @@
+# Copyright 2018 ForgeFlow S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo_test_helper import FakeModelLoader
+
+from odoo.exceptions import UserError
+from odoo.tests import common
+from odoo.tests.common import tagged
+
+from odoo.addons.base.tests.common import DISABLED_MAIL_CONTEXT
+
+
+@tagged("post_install", "-at_install")
+class TierTierValidation(common.TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, **DISABLED_MAIL_CONTEXT))
+        cls.loader = FakeModelLoader(cls.env, cls.__module__)
+        cls.loader.backup_registry()
+        from odoo.addons.base_tier_validation.tests.tier_validation_tester import (
+            TierValidationTester,
+        )
+
+        cls.loader.update_registry((TierValidationTester,))
+        cls.test_model = cls.env[TierValidationTester._name]
+
+        cls.tester_model = cls.env["ir.model"].search(
+            [("model", "=", "tier.validation.tester")]
+        )
+
+        # Access record:
+        cls.env["ir.model.access"].create(
+            {
+                "name": "access.tester",
+                "model_id": cls.tester_model.id,
+                "perm_read": 1,
+                "perm_write": 1,
+                "perm_create": 1,
+                "perm_unlink": 1,
+            }
+        )
+
+        # Create users:
```

Comment:
Is there a reason new users are created here?
(It comes at a cost, can the users from the base module be re-used?)

Reply:
Yeah, that will do, but we still need create one more users for tests case